lib/scripts.js

```diff
@@ -1,19 +1,34 @@
 let game;
+let player;
 
 function startNewGame () {
-  game = new Game()
+  game = new Game();
+  player = new Player();
+  createPlayers();
+  domUpdates.newGame();
+  game.createNewWheel();
 }
 
-// invoke startNewGame on 'Start' button
-startNewGame()
+function createPlayers() {
+  let playerOne = $('.player1-input').val();
+  let playerTwo = $('.player2-input').val();
+  let playerThree = $('.player3-input').val();
+  let player1 = new Player(playerOne);
+  let player2 = new Player(playerTwo);
+  let player3 = new Player(playerThree);
+  game.players.push(player1, player2, player3);
+}
 
```

Comment:
I see what you mean on this function in scripts. Does it belong in Player class or is it ok where it is. For functionality, I think we're fine but if told otherwise, we can refactor.

Reply:
This code is fine here but it should be refactored so that we dont have a bunch of variable names with numbers in them. You should have a common class between all these input elements, like `player-input`, select all elements with that class name, loop through them and use the index during that loop to replace the hard-coded 1s, 2s, and 3s.